demo/components/victory-area-demo.js

```diff
@@ -101,6 +101,8 @@ export default class App extends React.Component {
         <VictoryArea
           style={style} animate={{duration: 1000}}
           data={this.state.areaTransitionData}
+          title="What a great title!"
```

Comment:
Hmmm, is it possible for this to be more descriptive of the `<VictoryArea>` component to set a good example? 

`title="Animated Area Chart"` ? 
`desc="The height of the area animates over randomized data between 2 and 10."` ?